Chocolatey/tests/Unit/Private/Repair-PowerShellOutputRedirectionBug.tests.ps1

```diff
@@ -4,7 +4,7 @@ InModuleScope Chocolatey {
         Context 'Default' {
 
             It 'Should apply the fix silently' {
-                { Repair-PowerShellOutputRedirectionBug } | Should not throw
+                { Repair-PowerShellOutputRedirectionBug } | Should not Throw
```

Comment:
This one should be: `| Should -not -throw` iirc (for the latest version of Pester)